lib/client.js

```diff
@@ -217,12 +217,10 @@ Client.prototype._invoke = function(method, args, location, callback, options, e
   if (self.security && self.security.addOptions)
     self.security.addOptions(options);
 
-    if ((style === 'rpc')&& ( ( input.parts || input.name==="element" ) || args === null) ) {
-      assert.ok(!style || style === 'rpc', 'invalid message definition for document style binding');
-      message = self.wsdl.objectToRpcXML(name, args, alias, ns,(input.name!=="element" ));
-      (method.inputSoap === 'encoded') && (encoding = 'soap:encodingStyle="http://schemas.xmlsoap.org/soap/encoding/" ');
-    } else if (typeof (args) === 'string') {
-    message = args;
+  if ((style === 'rpc')&& ( ( input.parts || input.name==="element" ) || args === null) ) {
```

Comment:
could you please revert all formatting/intendation only changes in the files as otherwise it's really hard to tell what exactly was changed with your PR. thanks.